vagrant/provisioning/roles/alfresco7/tasks/main.yml

```diff
@@ -516,3 +516,8 @@
   retries: 100
   delay: 5
 
+- name: Set Java alternatives to OpenJDK 8
```

Comment:
Why this here, at the bottom and end of the play?

Reply:
Its because the new alfresco uses java 17 and once that is done the next roles utilize java 8 thats why I have used java 17 for alfresco and changed it to the default java once thats done

Reply:
> Its because the new alfresco uses java 17 and once that is done the next roles utilize java 8 thats why I have used java 17 for alfresco and changed it to the default java once thats done

And that java-17 is installed and declared as default in common role, right? And, from common role to alfresco7, we have a lot of roles in between. What if i use only upgrade tag? It will make java17 default, and i don't want java17 for ArkCase or anything else, only Alfresco7. Sorry for a lot of comments, i just want to have the whole picture of this PR.